jac-uk/admin
# Feat/2515 full application submitted

## What's included?
closes #2515 

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?

#### Staged exercises with open application after 18/06/2024 can see the new `full application submitted` status
- Go to the preview link: https://jac-admin-develop--pr2602-feat-2515-full-appli-k7d3ccp6.web.app/exercise/tzGyKNa4SakZ0CS8241V/stages/selection
- Submit the application of the exercise: https://jac-apply-develop--pr1258-feat-2515-full-appli-x7vkvrfi.web.app/vacancy/tzGyKNa4SakZ0CS8241V/
- Sign in with:
  ```
  application-0001@jac-dummy-email.jac
  1qazZAQ!1qaz
  ```
- The application should be set to `Full Application Submitted` status
- Set the application back to `Full Application Not Submitted`, it should move back to shortlisting 
- Click `Set status` button, it should include the new  `Full Application Submitted` status.
- Click `Show filters`, it should include the new  `Full Application Submitted` status.
- System should sent `Full Application Submitted` email to notify the candidate.

  https://github.com/user-attachments/assets/a58f97f4-dea5-466c-ad30-023a196b5fa6
  
  <img width="1153" alt="Screenshot 2024-11-12 at 08 04 32" src="https://github.com/user-attachments/assets/2308baf4-b769-418c-8864-26183c7b7d4c">

#### Staged exercises with open application before 18/06/2024 will not apply the new `full application submitted` status
- Go to the preview link: https://jac-admin-develop--pr2602-feat-2515-full-appli-k7d3ccp6.web.app/exercise/HD0kFoyONlPYSIwOH3tA/stages/selection
- Submit the application of the exercise: https://jac-apply-develop--pr1258-feat-2515-full-appli-x7vkvrfi.web.app/vacancy/HD0kFoyONlPYSIwOH3tA/
- Sign in with:
  ```
  application-0001@jac-dummy-email.jac
  1qazZAQ!1qaz
  ```
  
- The application should use the original `Shortlisting Passed` status.
- Click `Set status` button, it should not include the new  `Full Application Submitted` status.
- Click `Show filters`, it should not include the new  `Full Application Submitted` status.
 
  https://github.com/user-attachments/assets/21daa0d8-71a9-41a4-9ed8-2ad8efdd54c0

## Risk - how likely is this to impact other areas?
🟠 Medium risk - this does change code that is shared with other areas

## Additional context
Include screen grabs, video demo, notes etc.

## Related permissions
Have permissions been considered for this functionality?
- No permission changes required
- Permissions have been added / updated. Details:

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
